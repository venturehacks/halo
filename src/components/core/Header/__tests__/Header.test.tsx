import { render } from 'enzyme';
import React from 'react';

import { Header } from '../index';

describe('Header', () => {
  test('smoke', () => {
    const component = render(<Header h1>Play Tetris</Header>);
    expect(component.text()).toContain('Play Tetris');
  });

  describe('snapshots', () => {
    test(`h3 contrast AA`, () => {
      const component = render(
        <Header contrast="AA" h3>
          AA
        </Header>,
      );

      expect(component).toMatchSnapshot();
    });
  });
});
