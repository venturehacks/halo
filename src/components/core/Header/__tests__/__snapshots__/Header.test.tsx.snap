// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshots h3 contrast AA 1`] = `
<h3
  class="component flow __halo_textContrast_dark_AA __halo_fontSizeMap_size--xl __halo_fontWeight_medium"
>
  AA
</h3>
`;
